### ISSUE TITLE
Please change bugtracker link

### ISSUE THREAD

**Issue by pcav** [RuizGeo/Sampling#2]
It currently points to the repo, not to the issues page.
Thanks.

**Comment by RuizGeo** [comment]
tracker=https://github.com/RuizGeo/Sampling/issues

**Comment by pcav** [comment]
Exactly. Please chnage the metadata and the http://plugins.qgis.org/plugins/sampling/ page

**Comment by RuizGeo** [comment]
I solved the problem
